srcpkgs/wezterm/template

```diff
@@ -0,0 +1,73 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220905
+revision=1
+_srcver=20220905-102802-7d4b8249
+wrksrc="${pkgname}-${_srcver}"
+build_style=cargo
+make_check_args="--workspace --locked -- --skip e2e::sftp"
+hostmakedepends="pkg-config python3"
+makedepends="nautilus-python fontconfig-devel freetype-devel harfbuzz-devel
+ libssh2-devel libX11-devel libxkbcommon-devel libxkbcommon-x11
+ python3-devel wayland-devel xcb-util-devel xcb-util-image-devel
+ xcb-util-keysyms-devel xcb-util-wm-devel zlib-devel"
+depends="fonts-roboto-ttf"
+short_desc="GPU-accelerated cross-platform terminal emulator and multiplexer"
+maintainer="Joseph Benden <joe@benden.us>"
+license="MIT"
+homepage="https://wezfurlong.org/wezterm/"
+distfiles="https://github.com/wez/wezterm/releases/download/${_srcver}/${pkgname}-${_srcver}-src.tar.gz"
+checksum="5898af2bb2dbedcae2648764d5b7abd3d98b0aa3d05d171b09e0e3f76b7dd545"
+python_version=3
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	# The platform has 4 bugs in processing emotes; this shouldn't effect everybody?!
+	make_check_args+=" --skip cell::test::teeny_string"
+	make_check_args+=" --skip cell::test::test_width"
+	make_check_args+=" --skip surface::line::test::hyperlinks"
+	make_check_args+=" --skip surface::test::double_width"
+fi
+
+do_check() {
+	: ${make_cmd:=cargo}
+
+	# cannot use build_style cargo do_check because of --release flag.
+	${make_cmd} test --target ${RUST_TARGET} ${configure_args} \
+		${make_check_args}
```

Comment:
```suggestion
	# cannot use build_style cargo do_check because of --release flag.
	cargo test --target ${RUST_TARGET} ${make_check_args}
```

I would also remove make_check_args and include the condition here in do_check